ksqldb-engine/src/test/java/io/confluent/ksql/internal/StorageUtilizationMetricsReporterTest.java

```diff
@@ -54,17 +53,17 @@
 
   @Before
   public void setUp() {
-    listener = new StorageUtilizationMetricsReporter(metrics);
+    listener = new StorageUtilizationMetricsReporter();
+    listener.configure(
+        ImmutableMap.of(
+            KsqlConfig.KSQL_INTERNAL_METRICS_CONFIG, metrics
+        )
+    );
     when(metrics.metricName(any(), any(), (Map<String, String>) any())).thenAnswer(
       a -> new MetricName(a.getArgument(0), a.getArgument(1), "", a.getArgument(2)));
     StorageUtilizationMetricsReporter.setTags(BASE_TAGS);
   }
 
-  @After
-  public void cleanup() {
-    StorageUtilizationMetricsReporter.reset();
```

Comment:
I don't remember if I pointed it out before, but you'll notice lots of tests had `initialize()` and `reset()` calls that were an attempt to fix static registry clashes. None of those are needed anymore.